docs/docs/css-modules.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: Component-Scoped Styles with CSS Modules
+title: Estilos en Ámbito por Componentes con Módulos CSS
```

Comment:
Hmmmm no se si dejaría esto en el idioma original o si lo pondría como "Estilos con scope en Componentes con CSS Modules". ¿Qué opinan @xmartinezpujol @Darking360 @horacioh?

Reply:
Creo que yo lo pondría como: `Estilos isolados para componentes con Módulos CSS` 🤔 @gusma @xmartinezpujol @horacioh @nucliweb

Reply:
Hi, all. What would be the final translation for this?

Reply:
¿Cuál es la decisión final con respecto a la traducción de esto?